models/labels/validators/labels_validators_bnb.sql

```diff
@@ -26,10 +26,35 @@ FROM (VALUES
+    (array('bnb'),'0xD1d6bF74282782B0b3eb1413c901D6eCF02e8e28', 'BNB Validator: Richard Feynman', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0x733fdA7714a05960B7536330Be4DBB135bef0Ed6', 'BNB Validator: Legend IV', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0x4ee63a09170C3f2207aeCa56134Fc2Bee1b28e3C', 'BNB Validator: Sky', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0x73564052D8e469ED0721c4e53379DC3C91228930', 'BNB Validator: BitStack-0', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0x2f7bE8361C80A4c1e7e9aAF001d0877F1CFdE218', 'BNB Validator: raptas', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0x9ef9f4360c606c7AB4db26b016007d3ad0aB86a0', 'BNB Validator: nozti', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0xc2Be4EC20253B8642161bC3f444F53679c1F3D47', 'BNB Validator: tiollo', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0xEe01C3b1283AA067C58eaB4709F85e99D46de5FE', 'BNB Validator: glorin', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0x35E7a025f4da968De7e4D7E4004197917F4070F1', 'BNB Validator: piececoin', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0xd6caA02BBebaEbB5d7e581e4B66559e635F805fF', 'BNB Validator: leapbnb', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0x68Bf0B8b6FB4E317a0f9D6F03eAF8CE6675BC60D', 'BNB Validator: stakepulse', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0x8c4D90829CE8F72D0163c1D5Cf348a862d550630', 'BNB Validator: Everest', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0x82012708DAfC9E1B880fd083B32182B869bE8E09', 'BNB Validator: Kita', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0x6BBad7Cf34b5fA511d8e963dbba288B1960E75D6', 'BNB Validator: Ararat', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0x22B81f8E175FFde54d797FE11eB03F9E3BF75F1d', 'BNB Validator: Aconcagua', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0x78f3aDfC719C99674c072166708589033e2d9afe', 'BNB Validator: nariox', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0x29a97C6EfFB8A411DABc6aDEEfaa84f5067C8bbe', 'BNB Validator: C.R.E.A.M.', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    (array('bnb'),'0x2b3A6c089311b478Bf629C29D790A7A6db3fc1b9', 'BNB Validator: Figment', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())),
+    AS x (blockchain, address, name, category, contributor, source, created_at, updated_at)
```

Comment:
Syntax fix
```suggestion
    (array('bnb'),'0x6488Aa4D1955Ee33403f8ccB1d4dE5Fb97C7ade2', 'BNB Validator: Seoraksan', 'validators', 'soispoke', 'static', timestamp('2022-10-07'), now()),
    (array('bnb'),'0xCc8E6d00C17eB431350C6c50d8b8F05176b90b11', 'BNB Validator: Figment Inc.', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x61Dd481A114A2E761c554B641742C973867899D3', 'BNB Validator: Synclub', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x0BAC492386862aD3dF4B666Bc096b0505BB694Da', 'BNB Validator: Claude Shannon', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0xb218C5D6aF1F979aC42BC68d98A5A0D796C6aB01', 'BNB Validator: Alan Turing', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0xb4dd66D7c2C7E57F628210187192fb89d4b99dD4', 'BNB Validator: Saturn', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0xFE6E72b223f6d6Cf4edc6bFf92f30e84b8258249', 'BNB Validator: Hallasan', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x9bB832254BAf4E8B4cc26bD2B52B31389B56E98B', 'BNB Validator: stake2me', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0xD1d6bF74282782B0b3eb1413c901D6eCF02e8e28', 'BNB Validator: Richard Feynman', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x733fdA7714a05960B7536330Be4DBB135bef0Ed6', 'BNB Validator: Legend IV', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x4ee63a09170C3f2207aeCa56134Fc2Bee1b28e3C', 'BNB Validator: Sky', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x73564052D8e469ED0721c4e53379DC3C91228930', 'BNB Validator: BitStack-0', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x2f7bE8361C80A4c1e7e9aAF001d0877F1CFdE218', 'BNB Validator: raptas', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x9ef9f4360c606c7AB4db26b016007d3ad0aB86a0', 'BNB Validator: nozti', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0xc2Be4EC20253B8642161bC3f444F53679c1F3D47', 'BNB Validator: tiollo', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0xEe01C3b1283AA067C58eaB4709F85e99D46de5FE', 'BNB Validator: glorin', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x35E7a025f4da968De7e4D7E4004197917F4070F1', 'BNB Validator: piececoin', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0xd6caA02BBebaEbB5d7e581e4B66559e635F805fF', 'BNB Validator: leapbnb', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x68Bf0B8b6FB4E317a0f9D6F03eAF8CE6675BC60D', 'BNB Validator: stakepulse', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x8c4D90829CE8F72D0163c1D5Cf348a862d550630', 'BNB Validator: Everest', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x82012708DAfC9E1B880fd083B32182B869bE8E09', 'BNB Validator: Kita', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x6BBad7Cf34b5fA511d8e963dbba288B1960E75D6', 'BNB Validator: Ararat', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x22B81f8E175FFde54d797FE11eB03F9E3BF75F1d', 'BNB Validator: Aconcagua', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x78f3aDfC719C99674c072166708589033e2d9afe', 'BNB Validator: nariox', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x29a97C6EfFB8A411DABc6aDEEfaa84f5067C8bbe', 'BNB Validator: C.R.E.A.M.', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now()),
    (array('bnb'),'0x2b3A6c089311b478Bf629C29D790A7A6db3fc1b9', 'BNB Validator: Figment', 'validators', 'kartod', 'static', timestamp('2023-01-31'), now())
    ) AS x (blockchain, address, name, category, contributor, source, created_at, updated_at)

```

Reply:
@0xKARTOD would you mind applying this fix to your branch? after that we should be good to merge :)